src/scripts/gym/Gym.ts

```diff
@@ -1104,9 +1105,11 @@ gymList['Snowbelle City'] = new Gym(
     BadgeEnums.Iceberg,
     52000,
     'Impressive! Your Pokémon fought with great courage. I can tell that you\'ve trained your Pokémon well.',
-    [new ClearDungeonRequirement(1, GameConstants.getDungeonIndex('Pokémon Village'))]
+    [new ClearDungeonRequirement(1, GameConstants.getDungeonIndex('Pokémon Village'))],
+    () => {
+        App.game.quests.getQuestLine('The great hunt').beginQuest();
```

Comment:
Shouldn't this be `The Great Vivillon Hunt!`?

Reply:
Yes. I'm very sorry. I overlooked it and forgot to change the Name there.
I did it now. 😭